testsuite/src/it/java/io/quarkus/ts/startstop/utils/WhitelistLogLines.java

```diff
@@ -33,12 +33,20 @@
     FULL_MICROPROFILE(new String[]{
             // Some artifacts names...
             "maven-error-diagnostics",
-            "errorprone"
+            "errorprone",
+            // Needs fixing in the demo app?
+            "TestSecureController.java",
     }),
     GENERATED_SKELETON(new String[]{
             // It so happens that the dummy skeleton tries to find Mongo. This is expected.
             // See app-generated-skeleton/README.md for explanation of the scope.
             "The remote computer refused the network connection",
+            // Harmless warning
+            "The Agroal dependency is present but no JDBC datasources have been defined",
+            // This might be too generous. We might actually want to know about a property suddenly becoming unrecognized...
+            "Unrecognized configuration key",
+            // Hmm, weird, right? Deprecations should be fixed
```

Comment:
yup, Reactive Messaging is not yet Mutiny based but Quarkus warn about API usage :/
Info in https://issues.redhat.com/browse/QUARKUS-82